core/tools/dnsmasq/dnsmasq.conf

```diff
@@ -0,0 +1,38 @@
+# DHCP Server interface
+interface=eth0
+
+# IPv4 lease range
+dhcp-range=192.168.2.100,192.168.2.200,255.255.255.0,24h
+
+# IPv4 gateway (this device)
+dhcp-option=option:router,192.168.2.2
+
+# IPv6 lease range
+dhcp-range=::1681:ff,::1681:ffff,constructor:eth0,ra-names,slaac,24h
```

Comment:
Mixed feelings about enabling ipv6, in the end it only results in problems, I think that would be more sane to just disable it.

Reply:
Can you test without it?

I'm not a DHCP expert, so I don't know if this is a regular behavior, but for my machine, not having a IPv6 provided when the machine asked for it would prevent also the IPv4 to be provided.

Reply:
I just removed and appears to be working.

Reply:
Will check if there's anything on my machine regarding this configuration and test with another one.

Reply:
On tests, although I can see the message on failure to provide an IPv6, this time the IPv4 got leased, so I assume it was a coincidence what I have seem previously. 
Removing.